package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "d2l-telemetry-browser-client",
-  "version": "0.0.1",
```

Comment:
Can't we still remove the prebuild, build steps? and the test:bundle stuff? as we no longer need it for deploying to the CDN?